test/ui/ResultActionsMenuTest.ts

```diff
@@ -0,0 +1,44 @@
+import * as Mock from '../MockEnvironment';
+import { IQueryResult } from '../../src/rest/QueryResult';
+import { FakeResults } from '../Fake';
+import { ResultActionsMenu } from '../../src/ui/ResultActions/ResultActionsMenu';
+import { $$ } from '../../src/utils/Dom';
+
+export function ResultActionsMenuTest() {
+  describe('ResultActionsMenu', () => {
+    let resultActions: ResultActionsMenu;
+    let test: Mock.IBasicComponentSetup<ResultActionsMenu>;
+    let fakeResult: IQueryResult;
+
+    beforeEach(() => {
+      fakeResult = FakeResults.createFakeResult();
```

Comment:
You could remove all this and simply do : 

`test = Mock.basicResultComponentSetup<ResultActionMenu>(ResultActionMenu);`